internal/backend/crypto/age/ssh.go

```diff
@@ -69,6 +66,24 @@ func (a *Age) getSSHIdentities(ctx context.Context) (map[string]age.Identity, er
 	return ids, nil
 }
 
+func getSSHDir() (string, error) {
+	preferredPath := os.Getenv("GOPASS_SSH_DIR")
+	sshDir := filepath.Join(preferredPath, ".ssh")
+	if fsutil.IsDir(sshDir) && preferredPath != "" {
```

Comment:
Could you swap both conditions?
`os.Getenv` is most likely to return an empty string for most users in most of the case, so let's not run `IsDir` if we don't need to.